tests/integration/cocoon/test_seed_sample_data.py

```diff
@@ -0,0 +1,204 @@
+import unittest
+from pathlib import Path
+import pandas as pd
+import lusid
+from lusidtools.cocoon.seed_sample_data import seed_data
+from lusidtools.cocoon.utilities import load_json_file
+from lusidtools.cocoon.utilities import create_scope_id
+import logging
+import os
+import datetime
+import pytz
+import random
+
+logger = logging.getLogger()
+
+
+class CocoonTestSeedDataSetupTests(object):
```

Comment:
rename to `CocoonSeedDataTestsBase`

its probably worth commenting about how its used i.e. need to derive and set up data in the derived class (just thinking what a client would do if they come across this)